t/features/regex.t

```diff
@@ -19,25 +19,33 @@ ensure-feature-flag("REGEX");
 
     outputs $program, "True\n", "Regexes are truthy";
 }
-
-{
-    my $program = q:to/./;
-        say(/"Blofeld"/.fullmatch("Blofeld"));
-        say(/"Blofeld"/.fullmatch("Bond"));
-        say(/"Blofeld"/.fullmatch("this is Blofeld's work, I know it"));
-        .
-
-    outputs $program, "True\nFalse\nFalse\n", "the .fullmatch method matches the whole string";
-}
-
 {
-    my $program = q:to/./;
-        say(/"Blofeld"/.search("Blofeld"));
-        say(/"Blofeld"/.search("Bond"));
-        say(/"Blofeld"/.search("this is Blofeld's work, I know it"));
-        .
-
-    outputs $program, "True\nFalse\nTrue\n", "the .search method matches part of the string";
+    my %programs =
+        'say(/"abc" "def"/.fullmatch("abcdef"));' => True,
+        'say(/"abc" "def"/.fullmatch("abcde"));' => False,
+        'say(/"abc"? "def"?/.fullmatch(""));' => True,
+        'say(/"abc"? "def"?/.fullmatch("abc"));' => True,
+        'say(/"abc"? "def"?/.fullmatch("abcdef"));' => True,
+        'say(/"abc"? "def"?/.fullmatch("def"));' => True,
+        'say(/"abc"? "def"?/.fullmatch("defxyz"));' => False,
+        'say(/"abc" "def"+/.fullmatch("abc"));' => False,
+        'say(/"abc" "def"+/.fullmatch("abcdef"));' => True,
+        'say(/"abc" "def"+/.fullmatch("abcdefdefdef"));' => True,
+        'say(/"abc" ["def"+ "xyz"]/.fullmatch("abcdefdefdefxyz"));' => True,
+        'say(/"abc" ["def"+ "xyz"]/.fullmatch("abcdefdefdefxy"));' => False,
+        'say(/"abc" ["def"* "xyz"? "|"]+/.fullmatch("abc|"));' => True,
+        'say(/"abc" ["def"* "xyz"? "|"]+/.fullmatch("abcxyz|"));' => True,
+        'say(/"abc" ["def"* "xyz"? "|"]+/.fullmatch("abcdef|"));' => True,
+        'say(/"abc" ["def"* "xyz"? "|"]+/.fullmatch("abcdefdefdefxyz|defdef|"));' => True,
+        'say(/"abc" ["def"* "xyz"? "|"]+/.fullmatch("abcdefdefdefxy|"));' => False,
+        'say(/"abc" ["def" | "xyz"]+/.fullmatch("abcdefxyzdef"));' => True,
+        'say(/"abc" ["def" | "xyz"]+/.fullmatch("abcdefxyzde"));' => False,
+        'say(/"abc" ["def" | "xyz"]+/.fullmatch("abcdefxyzxydef"));' => False,
```

Comment:
These are all tests for `.fullmatch`. It looks like the tests for `.search` were removed and not replaced by new tests...?